FirebaseSegmentation.podspec

```diff
@@ -0,0 +1,42 @@
+Pod::Spec.new do |s|
+  s.name             = 'FirebaseSegmentation'
+  s.version          = '0.1.0'
+  s.summary          = 'Firebase Segmentation SDK'
+  s.description      = <<-DESC
+Firebase Segmentation enables you to associate your custom application instance ID with Firebase for user segmentation.
+                       DESC
+
+  s.homepage         = 'https://firebase.google.com'
+  s.license          = { :type => 'Apache', :file => 'LICENSE' }
+  s.authors          = 'Google, Inc.'
+  s.source           = {
+    :git => 'https://github.com/firebase/firebase-ios-sdk.git',
+    :tag => 'Segmentation-' + s.version.to_s
+  }
+
+  s.ios.deployment_target = '8.0'
+  s.cocoapods_version = '>= 1.4.0'
+  s.static_framework = true
+  s.prefix_header_file = false
+
+  s.source_files = 'FirebaseSegmentation/Sources/**/*.[mh]'
+  s.public_header_files = 'FirebaseSegmentation/Sources/Public/*.h'
+
+  s.dependency 'FirebaseCore', '~> 6.7'
+  s.dependency 'FirebaseInstanceID', '~> 4.3'
```

Comment:
InstanceID is deprecated and should not be used in a new podspec.

Reply:
I think the FIS migration work for segmentation is being done in https://github.com/firebase/firebase-ios-sdk/pull/5622

Reply:
Thanks for the comment.

Actually #5622 haven't completed yet. I'll address all comments from that pr and this pr together then merge this one.